salt/grains/core.py

```diff
@@ -448,7 +448,7 @@ def _osx_memdata():
     sysctl = salt.utils.path.which('sysctl')
     if sysctl:
         mem = __salt__['cmd.run']('{0} -n hw.memsize'.format(sysctl))
-        swap_total = __salt__['cmd.run']('{0} -n vm.swapusage'.format(sysctl)).split()[2]
+        swap_total = __salt__['cmd.run']('{0} -n vm.swapusage'.format(sysctl)).split()[2].replace(',', '.')
```

Comment:
Is this due to the locale?

Here you can find a more generic approach:
https://stackoverflow.com/questions/13362121/convert-python-strings-into-floats-explicitly-using-the-comma-or-the-point-as-se

I wonder if there are no better ways to do it though.

Reply:
yeah looks like you are right that this is related to locale. let me look into a better fix. thanks :)